docs/development/contributing/index.md

```diff
@@ -228,6 +228,7 @@ From the `py-polars` directory, run `make fmt` to make sure your additions pass
 Polars uses Sphinx to build the API reference.
 This means docstrings in general should follow the [reST](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) format.
 If you want to build the API reference locally, go to the `py-polars/docs` directory and run `make html SPHINXOPTS=-W`.
+By default, this uses only one thread. To speed it up, run `make html SPHINXOPTS="-W -j auto"` to use all available cpus.
```

Comment:
This is enabled by default - see the Makefile.

Reply:
Looks like the Makefile sets the following by default, as you said:

https://github.com/pola-rs/polars/blob/f6ba24231ac85f8f793431c8dab434e14f53c36f/py-polars/docs/Makefile#L8

However, if you do as the contrib guide suggests and run `make html SPHINXOPTS=-W`, then you will override that default and get rid of the `-j auto`. So we should change the guide to say `make html`.

Reply:
You're right!